modules/kontraktor-http/src/main/java/org/nustaq/utils/TrafficMonitorUtil.java

```diff
@@ -0,0 +1,22 @@
+package org.nustaq.utils;
```

Comment:
I wanted to re-use that code in other Connectors like UndertowWebsocketServerConnector without creating a common parent class, etc.